modules/working-with-mirrored-images.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * builds/creating-build-inputs.adoc
+
+:_content-type: CONCEPT
+
+[id="working-with-mirrored-images_{context}"]
+= Configuring pull secrets for mirror registry
+
+If run a build that pulls an image from a mirrored registry and get a `build error: failed to pull image` message, you can resolve the error either of the following methods:
+
+* Set the pull secret on the build config object.
+* Create a secret containing auth credentials for the builder image’s repository and all known mirrors. In this case, create a pull secret for creds to docker.io and its mirrors.
```

Comment:
I would flip the order here - create the secret, then reference the secret in the BuildConfig:

```suggestion
* Create a secret containing auth credentials for the builder image’s repository and all known mirrors. In this case, create a pull secret for creds to docker.io and its mirrors.
* Use this secret as the pull secret on the build config object.

```

Reply:
Updated. Thanks.